CodeModel/src/main/java/org/openzen/zenscript/codemodel/compilation/impl/capture/LocalVariableExpression.java

```diff
@@ -23,7 +23,8 @@ public LocalVariableExpression(CodePosition position, CompilingVariable variable
 
 	@Override
 	public LocalExpression capture(LambdaClosure closure) {
-		CapturedExpression value = new CapturedLocalVariableExpression(position, variable.eval(), closure);
+		CapturedLocalVariableExpression value = new CapturedLocalVariableExpression(position, variable.eval(), closure);
+		closure.add(value);
```

Comment:
💭 Thought: Should we add the `closure.add(value)` call to CapturedExpression's Constructor so that it isn't forgotten (like it was in here)?